iroha/src/sumeragi.rs

```diff
@@ -1691,19 +1691,62 @@ mod tests {
     use {
         crate::{config::Configuration, init, maintenance::System, queue::Queue, torii::Torii},
         async_std::{prelude::*, sync, task},
+        network::*,
         std::time::Duration,
     };
 
     #[cfg(feature = "network-mock")]
-    const CONFIG_PATH: &str = "config.json";
-    #[cfg(feature = "network-mock")]
-    const BLOCK_TIME_MS: u64 = 1000;
-    #[cfg(feature = "network-mock")]
-    const COMMIT_TIME_MS: u64 = 1000;
-    #[cfg(feature = "network-mock")]
-    const TX_RECEIPT_TIME_MS: u64 = 200;
+    mod network {
+        pub const CONFIG_PATH: &str = "config.json";
+        pub const BLOCK_TIME_MS: u64 = 1000;
+        pub const COMMIT_TIME_MS: u64 = 1000;
+        pub const TX_RECEIPT_TIME_MS: u64 = 200;
+        pub const TRANSACTION_TIME_TO_LIVE_MS: u64 = 100_000;
+    }
+
     #[cfg(feature = "network-mock")]
-    const TRANSACTION_TIME_TO_LIVE_MS: u64 = 100_000;
+    fn wsv_create_account(
+        wsv: &mut WorldStateView,
+        genesis_key_pair: &KeyPair,
+        account_name: &str,
+        domain_name: &str,
+    ) -> KeyPair {
+        let create_domain =
+            RegisterBox::new(IdentifiableBox::Domain(Domain::new(domain_name).into()));
+        let key_pair = KeyPair::generate().expect("Failed to generate KeyPair.");
+
+        let create_account = RegisterBox::new(IdentifiableBox::Account(
+            Account::with_signatory(
+                AccountId::new(account_name, domain_name),
+                key_pair.public_key.clone(),
+            )
+            .into(),
+        ));
+
+        let pending_block = PendingBlock::new(vec![AcceptedTransaction::from_transaction(
+            Transaction::new(
+                vec![create_domain.into(), create_account.into()],
+                <Account as Identifiable>::Id::genesis_account(),
+                TRANSACTION_TIME_TO_LIVE_MS,
+            )
+            .sign(genesis_key_pair)
+            .expect("Failed to sign."),
+            4096,
+        )
+        .expect("Failed to accept tx.")]);
+
+        let topology = NetworkTopology::new(&wsv.world.trusted_peers_ids, Some(Hash([0u8; 32])), 1)
+            .init()
```

Comment:
It is better not to take a mutable wsv and apply instructions to it, but to `WSV::new(World::new())` and in new `World` setup all the needed data as in queue. Just add there a root@global account and use this wsv for all peers in all tests. No more changes to sumeragi tests are necessary here.

Reply:
Take a look at `queue::tests::world_with_test_domains`